clients/vscode/package.json

```diff
@@ -69,6 +69,10 @@
       {
         "command": "tabby.notifications.resetMuted",
         "title": "Tabby: Reset notifications marked as \"Don't Show Again\""
+      },
+      {
+        "command": "tabby.ai.explainCodeBlock",
```

Comment:
I think it's better to use `tabby.chat` as prefix: `tabby.chat.explainCodeBlock`

Reply:
`tabby.experimental.chat.explainCodeBlock`